lib/bundlex/build_script.ex

```diff
@@ -55,6 +55,14 @@ defmodule Bundlex.BuildScript do
     end
   end
 
+  # TODO: this is obviously awkward but I'm very tired.
+  # should the toolchains define these?
+  defp join_commands_run(commands, :windows) do
+    Enum.join(commands, " && ")
+  end
+
+  defp join_commands_run(commands, family), do: join_commands(commands, family)
```

Comment:
Wouldn't `Enum.join(commands, " && \r\n")` or `Enum.join(commands, " && \\\r\n")` work? Otherwise, I guess it's fine to do `Enum.join(commands, " && ")` in all cases ;)

Reply:
> Wouldn't` Enum.join(commands, " && \r\n") or Enum.join(commands, " && \\\r\n") work?

I actually don't think so. In a batch script a trailing `&&` doesn't work.

```bat
rem foo.bat
@echo off
echo foo &&
echo bar
```

Trying to run this outputs `The syntax of the command is incorrect.`

> Otherwise, I guess it's fine to do Enum.join(commands, " && ") in all cases ;)

I could do this, I think. It makes the stored script more awkward, but that's fine.